gcloud/_apitools/util.py

```diff
@@ -78,9 +78,6 @@ def AcceptableMimeType(accept_patterns, mime_type):
 
     def MimeTypeMatches(pattern, mime_type):
         """Return True iff mime_type is acceptable for pattern."""
-        # Some systems use a single '*' instead of '*/*'.
```

Comment:
Dead code:  the containing function, `AcceptableMimeType`, raises an exception for any pattern which doesn't contain a slash, before ever clalling `MimeTypeMatches`.